app/subscriber/src/features/my-reports/edit/ReportEditContext.tsx

```diff
@@ -60,8 +60,6 @@ export interface IReportEditContext {
   setPreviewLastUpdatedOn: React.Dispatch<React.SetStateAction<string | undefined>>;
   testData: IReportInstanceContentModel[];
   regenerateTestData: () => void;
-  showReportData: boolean;
```

Comment:
Using the test data is now controlled by the component so that changing the view doesn't change for all charts.